compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1758,6 +1758,9 @@ error.unsupported.type.intersection=\
 error.wild.card.binding.pattern.only.supports.type.any=\
   a wildcard binding pattern can be used only with a value that belong to type ''any''
 
+error.configurable.variable.module.ambiguity.found=\
+  configurable variable name ''{0}'' creates ambiguity with a module
```

```suggestion
  configurable variable name ''{0}'' creates an ambiguity with a module
```

---

And this is always from an imported module, right? How about

```suggestion
  configurable variable name ''{0}'' creates an ambiguity with an imported module
```

or if we are adding the name
```suggestion
  configurable variable name ''{0}'' creates an ambiguity with imported module ''{1}''
```